### DISCUSSION TITLE
Pickup Time not showing in order details nor emails anymore

### DISCUSSION THREAD

**Discussion by supra48** [mattbanks/woocommerce-local-pickup-time#198] [Q&A]
Hello, I am very grateful for this plugin but lately, a core functionality stopped working on my website.

I am not sure what went wrong here, but now the pickup time does not show in the order details or order received emails. I tried disabling all plugins, kept only Local Pickup Time and WooCommerce, and same issue. What do you recommend I do here?
Thank you

**Comment by timnolte** [comment]
Did this start when you updated to WordPress 6.0 or 6.1, or even a recent update to WooCommerce? There have been some significant changes in both recently and I can't say that I have tested completely with them all. I did recently setup a brand new test site and things appeared to be working as expected. Additionally, I haven't fully tested the plugin on PHP 8, which as I've seen on a lot of other WordPress sites in general having issues with PHP 8. Without more details on the circumstances of when it stopped working it's hard to pinpoint the issue.

**Reply by supra48** [reply]
Hello,

Thank you so much for the quick reply. I did a lot of changes lately to the website, including upgrading WooCommerce yes, that make it hard to track what exactly affected the plugin.
The steps I took to try and isolate the issue are:
I disabled all other plugins -> the issue persisted
I downgraded to WordPress 5.9.3 -> the issue persisted

My PHP version is 7.4.32

I will try to downgrade WooCommerce and see what happens.

**Reply by supra48** [reply]
I checked the DB, inside the wp_postmeta table, the key _local_pickup_time_select is not filled for recent orders. It is there for orders that were working before.
So the problem is that the value is not inserted in the DB.
I also enabled debug mode, and the debug.log doesn't contain any errors. Should I look elsewhere?

**Reply by supra48** [reply]
@timnolte, did you have time to check my comments please?

**Reply by timnolte** [reply]
@supra48 no I haven't had an opportunity to look into this further yet. Also another question, did you migrate to the new custom order tables?

**Reply by supra48** [reply]
I haven't upgraded to the new custom order tables. Should I try?

**Reply by timnolte** [reply]
@supra48 no I haven't fully tested against that yet so want to keep that out of the equation.
